share/completions/pygmentize.fish

```diff
@@ -1,8 +1,12 @@
 function __fish_print_pygmentize
-    set -l lines (pygmentize -L $argv[1] | string match -r '^(?:\* |    ).*(?:)$' | string replace -r '\* (.*):$' '$1' | string replace -r '^(.*).$' '$1' | string trim)
+    set -l lines (pygmentize -L $argv[1] | string match -r '^(?:\* |    ).*(?:)$' | string replace -r '\* (.*):$' '$1' | string replace -r '^(.*)^\.' '$1' | string trim)
```

Comment:
This isn't massively important either way. I'm reasonably sure we have completions with periods in the description.